File: src/main/java/com/p2p/lending/dao/TradeDao.java
package com.p2p.lending.dao;

import java.util.List;
import java.util.Map;

import com.p2p.lending.pojo.Trade;



public interface TradeDao {
	List<Trade> selecttd(Map<String, Object> map);
	int tradecount(Map<String, Object> map);
	boolean insertT(Trade td);
	
	List<Trade> selectMoney(Integer uid);
}

File: src/main/java/com/p2p/lending/dao/TradeDao.kt
package com.p2p.lending.dao

import com.p2p.lending.entity.Trade


import org.springframework.stereotype.Repository

@Repository
interface TradeDao {

    fun selecttd(map: Map<String, Any?>): List<Trade>

    fun tradecount(map: Map<String, Any?>): Int

    fun insertT(td: Trade): Boolean

    fun selectMoney(uid: Int?): List<Trade>
}
